modules/tinymce/src/core/test/ts/browser/annotate/AnnotationStylingTest.ts

```diff
@@ -97,7 +97,7 @@ describe('browser.tinymce.core.annotate.AnnotationStylingTest', () => {
 
       if (checkOtherNodes) {
         const parents = SelectorFilter.ancestors(elm, '*', (e) => Compare.eq(e, body));
-        const children = SelectorFilter.children(elm, '*');
+        const children = SelectorFilter.children(elm, '*:not(source)');
```

Comment:
I think a descriptive comment explaining why we are excluding `source` tags here might help the future code reader.

Reply:
Done